tests/Tests.Core/Extensions/EphemeralClusterExtensions.cs

```diff
@@ -63,8 +63,8 @@ public static ConnectionSettings CreateConnectionSettings<TConfig>(this IEphemer
 
 				var noCertValidation = current.ServerCertificateValidationCallback == null;
 
-				if (cluster.ClusterConfiguration.EnableSecurity && notAlreadyAuthenticated)
-					settings = settings.BasicAuthentication(ClusterAuthentication.Admin.Username, ClusterAuthentication.Admin.Password);
+				// if (notAlreadyAuthenticated)
+				// 	settings = settings.BasicAuthentication(ClusterAuthentication.Admin.Username, ClusterAuthentication.Admin.Password);
 				if (cluster.ClusterConfiguration.EnableSsl && noCertValidation)
```

Comment:
Can this be deleted?

Reply:
Updated in 149f3a7534df2ab066ac6e79a79588a698ab4091